### ISSUE TITLE
Multiple release branches and multiple release drafts

### ISSUE THREAD

**Issue by alvarosanchez** [release-drafter/release-drafter#404]
First of all, congratulations for this automation. I think it's pure gold.

Our use case in the [Micronaut team](https://github.com/micronaut-projects) is that we create releases not only from `master`, but also from branches such as `1.0.x`, `1.1.x`, etc. In this scenario, release-drafter overwrites whatever draft version it finds, and this isn't desirable in all cases.

Would it be possible to setup different release-drafter configurations so that we could have multiple release drafts at a given moment, and then release-drafter would filter the draft to update based on some regexp or similar?

Maybe this is related to #203 and #175

**Comment by alvarosanchez** [comment]
Unfortunately, we have moved to another approach to generate release notes.

**Comment by jetersen** [comment]
I think this is mostly solved by GitHub actions input and once #545 this should be achievable from any workflow and `config-name` input.

**Comment by peter-dolkens** [comment]
@jetersen can you confirm, having multiple configs will allow me to specify a `config-name` and the release drafted by that config won't interfere with releases drafted by another config?

In our scenario, we've got multiple tenants, so ideally we'll be able to set something up like:

branch: tenantA
version: major.minor.patch-tenantA

branch: tenantB
version: major.minor.patch-tenantB

**Comment by jetersen** [comment]
@peter-dolkens as long as the config with that `config-name` lives on the default branch it will work for now.

In the future, this will definitely change. 
I have a branch locally where I am trying to split the GitHub App (using probot) and GitHub Action functionality.

There I would need to implement a new method for getting the config since GitHub action will not be using the probot method `context.config`
Than I will properly have the GitHub app use the very same function to share that functionality of getting the config from the branch.

See #549

**Comment by agpiermarini** [comment]
@jetersen I have two workflows and two configs, one targeting `master` and the other `develop` (workflow and config files below). I would expect this to generate two draft releases and to manage them in parallel,  but a push to `master` creates a properly-named draft release that a subsequent push to `develop` will rename (versus drafting a new one with the appropriate suffix), and _vice versa_. Is this use case supported, and if so, do you have any suggestions for how to achieve the desired output?

The workflow and config files are as follows:

### Master
#### Workflow (`.github/workflows/release-drafter.yml`)
```
name: Release Drafter

on:
  push:
    # branches to consider in the event; optional, defaults to all
    branches:
      - master

jobs:
  update_release_draft:
    runs-on: ubuntu-latest
    steps:
      # Drafts your next Release notes as Pull Requests are merged into the above branches
      - uses: release-drafter/release-drafter@v5.11.0
        with:
          config-name: release-drafter.yml
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

#### Config (`.github/release-drafter.yml`)
```
name-template: "v$NEXT_PATCH_VERSION"
tag-template: "v$NEXT_PATCH_VERSION"
change-template: "- $TITLE (#$NUMBER, @$AUTHOR)"
template: |
  ## Changes
  $CHANGES

  ## Contributors
  $CONTRIBUTORS
```

### Develop
#### Workflow (`.github/workflows/release-drafter-develop.yml`)
```
name: Release Drafter

on:
  push:
    # branches to consider in the event; optional, defaults to all
    branches:
      - master

jobs:
  update_release_draft:
    runs-on: ubuntu-latest
    steps:
      # Drafts your next Release notes as Pull Requests are merged into the above branches
      - uses: release-drafter/release-drafter@v5.11.0
        with:
          config-name: release-drafter-develop.yml
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

#### Config (`.github/release-drafter-develop.yml`)
```
name-template: "v$NEXT_PATCH_VERSION-develop"
tag-template: "v$NEXT_PATCH_VERSION-develop"
change-template: "- $TITLE (#$NUMBER, @$AUTHOR)"
template: |
  ## Changes
  $CHANGES

  ## Contributors
  $CONTRIBUTORS
```

**Comment by mkurz** [comment]
I think you might be interested in this comment where I wrote a howto with details on how to generate different release drafts for different branches: https://github.com/release-drafter/release-drafter/issues/656#issuecomment-806184887
